Lib/asyncio/futures.py

```diff
@@ -152,11 +152,11 @@ def cancel(self, msg=None):
             return False
         self._state = _CANCELLED
         self._cancel_message = msg
-        self.__schedule_callbacks()
+        self._finish_execution()
         return True
 
-    def __schedule_callbacks(self):
-        """Internal: Ask the event loop to call all callbacks.
+    def _finish_execution(self):
```

Comment:
For clarification, what's the reason for this name change? This is still, effectively, scheduling all callbacks -- "finish execution" is a bit more ambiguous to me.

Reply:
Yes, in `Future`, "_schedule_callbacks" is perfectly fine to describe what the function does. When overriding this function in `Task`, I think `_finish_execution` is more meaningful to indicate that it will be called when the task completes.